src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/ValidationAttributeStore.cs

```diff
@@ -230,8 +232,62 @@ internal bool TryGetPropertyStoreItem(string propertyName, [NotNullWhen(true)] o
                     propertyStoreItems[property.Name] = item;
                 }
 
+                return CombineMetadataTypeAttributes(propertyStoreItems);
+            }
+
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:DynamicallyAccessedMembersAttribute",
+                       Justification = "The ctor is marked with DynamicallyAccessedMembers.")]
```

Comment:
This is not a valid justification for suppressing the trim warnings. The `Type` being passed into the ctor is not the same type that is being reflected upon here.

Reply:
Thank you. Working on it 😊